sympy/stats/frv.py

```diff
@@ -319,7 +320,7 @@ def domain(self):
     @property
     @cacheit
     def _density(self):
-        return dict((frozenset(((self.symbol, val),)), prob)
+        return dict((sympify(frozenset(((self.symbol, val),))), prob)
```

Comment:
Perhaps it would be possible to use simply ``FiniteSet(...)`` instead of the combination ``sympify(frozenset(...))``.